nms-15802 Add preselect auth to page in page sequence (#6384)
OpenNMS/opennms: docs/modules/reference/pages/service-assurance/monitors/PageSequenceMonitor.adoc

Colon-separated credentials such as USERNAME:PASSWORD.
| n/a

| preselect-auth

> | Preselect authentication method. At the time allows only "basic" what corresponds to Basic Authentication and requires that user name and password are provided in the attribute `user-info`

Reviewer: ```suggestion
| Preselect authentication method. 
When specified, allows only basic authentication.
User name and password must be provided in the `user-info` attribute.
```
do you mean "currently" when you say "at the time" or how I changed it. I'm also not clear if "basic" is a specific value.

WARNING: You need to delete existing RRD files and let them be recreated with the new list of data sources when you add a ds-name attribute to a page in a sequence that is already storing response-time data.

== Preselect authentication
When authentication is required, the first HTTP request receives a "401 Unauthorized" response with possible authentication methods. The second request is sent using the selected authentication method.

> Using the `preselect-auth` attribute allows you to use a specific authentication right on the first request.

Reviewer: ```suggestion
Using the `preselect-auth` attribute lets you to use a specific authentication on the first request.
```
I prefer "allows" but our style guide requires using "lets" instead of "allows" in this context.

> Some servers do not tell you which authentication methods are supported. In these cases, using the `preselect-auth` attribute is the only option.

Reviewer: ```suggestion
Some servers do not tell you which authentication methods they support. 
In these cases, using the `preselect-auth` attribute is the only option.
```
